react/ecmascript/01 - spread/index.js

```diff
@@ -0,0 +1,32 @@
+import { isArray } from './utils';
+
+export function min(expect, ...c) {
+  if (expect === undefined) {
+    return expect;
+  } else {
+      if (!isArray(expect)) {
```

Comment:
You can use ternary. For example: return isArray(expect) ? Math.min(...expect) : Math.min(expect, ...c)